metadata-service/war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -114,6 +120,10 @@
     <servlet-name>openapiServlet</servlet-name>
     <url-pattern>/openapi/*</url-pattern>
   </servlet-mapping>
+  <servlet-mapping>
+    <servlet-name>operationsServlet</servlet-name>
+    <url-pattern>/welcome/*</url-pattern>
```

Comment:
This should be an appropriate url pattern match (if it's needed, I think this call all live under the openAPI servlet).

Reply:
Oops I fixed this but forgot to commit it 😂